### ISSUE TITLE
Proper output format of results of queries using A, E, [], <>, sup, and inf.

### ISSUE THREAD

**Issue by GameMonkey** [UPPAALModelChecker/UPPAAL-Meta#96]
**Is your feature request related to a problem? Please describe.**
A proper output format for query results in, for example, JSON or CSV.

When running verifyta on the command line and one needs to parse the results of all the queries, the first step is the parse the test parsed to standard out. Having a structured output format, I could eliminate this parsing step with regex and whatnot.

*Note*: I am not talking about storing strategies and such, which I know can be stored. I am talking about queries using A, E, [], <>, sup, and inf.

**Describe the solution you'd like**
Any structured file that allows me to grasp the result of a query given its query number.

**Comment by AstridHornBrorholt** [comment]
I agree with this, and would especially also like an option to include which query each result corresponds to. This business of just stating it is "query 1" is often not much help when you have a 10+ line query file with comments mixed in. Here it is better to print the full query before printing the result. 

This should be possible for both standard user-readable output and the suggested JSON output
